frontend/src/components/CreatePeerDialog/CreatePeerDialog.js

```diff
@@ -1,65 +1,80 @@
-import React, {useState, useEffect} from 'react';
+import React, { useState } from 'react';
 import {
-  Paper,
-  IconButton,
-  Menu,
-  MenuItem,
-  Typography,
-  Grid,
-  Box,
-  Link,
+  Button,
   Dialog,
+  DialogActions,
   DialogTitle,
   DialogContent,
-  DialogContentText,
+  FormControlLabel,
+  Switch,
   TextField,
-  DialogActions,
-  Button,
-  FormControl,
-  InputLabel,
-  Select,
 } from "@material-ui/core";
-import { MoreVert as MoreIcon } from "@material-ui/icons";
-import {useHistory} from "react-router-dom";
-import classnames from "classnames";
+import { useHistory } from "react-router-dom";
 
 // styles
-import useStyles from "./styles";
-
-import Widget from "../../components/Widget";
-import SqueakThreadItem from "../../components/SqueakThreadItem";
+import {makeStyles} from '@material-ui/core/styles';
 
 import {
   createPeerRequest,
 } from "../../squeakclient/requests"
 
+const useStyles = makeStyles((theme) => ({
```

Comment:
Can the styles code be in a separate file somewhere else?

There used to be a file `frontend/src/components/CreatePeerDialog/styles.js`. I don't know if that is the right place.

But I don't really like having the style code in the middle of this file.